plugins/Observability/src/Observability.cpp

```diff
@@ -1850,10 +1838,41 @@ void Observability::showReport(bool b)
 	}
 }
 
+bool Observability::isMoon(QList<StelObjectP> & objectSelection)
+{
+   return !objectSelection.empty() && "Moon" == objectSelection[0]->getEnglishName();
+}
+
+bool Observability::isSun(QList<StelObjectP> & objectSelection)
+{
+   return !objectSelection.empty() && "Sun" == objectSelection[0]->getEnglishName();
+}
+
+bool Observability::isStar(QList<StelObjectP> & objectSelection)
+{
+   if (objectSelection.empty()) {
+       return true; // no object selected, so the selection is the screen/space. 
```

Comment:
The method should be named isStarOrNothing() in this case!

Reply:
Maybe even isNoSSO(), as it can refer to DSO as well.

Reply:
I'll update this. Does "SSO" stand for "Solar System Object?" What does "DSO" stand for?

Reply:
Yes, SSO=Solar System Object. DSO=Deep Sky Object (galaxy, cluster, nebula, SNR [SuperNova Remnant]...)